docker/Dockerfile-fargo-master

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.6
+
+ENV GOROOT=/usr/local/go
+
+RUN go get github.com/tools/godep
+RUN go get github.com/hudl/fargo
```

Comment:
Ok, this will not work on Docker for Windows yet ... Probably worth changing it later https://github.com/docker/docker/issues/22981